### ISSUE TITLE
Funding

### ISSUE THREAD

**Issue by mpereira** [immutable-js-oss/immutable-js#219]
Hi folks,

I appreciate the continued work on Immutable. To me it's still the only sane way to deal with data in JavaScript.

I was wondering if you have any thoughts regarding taking sponsors through GitHub (or some other way)? I'd be glad to contribute and I'm sure there are companies and other people invested in the project that could do the same. It would be great to have a group of folks maintaining this project in a sustainable way.

**Comment by Methuselah96** [comment]
Thanks for the issue! I went ahead and added a sponsor button. Development has been slow recently and getting some sponsors would definitely motivate me more to work on this, since currently this is all pro bono,

**Comment by mpereira** [comment]
No worries!

Maybe it makes sense creating an issue on the original repository too, and pinging Lee Byron to see if he can help with a shout out on Twitter or something.

**Comment by Methuselah96** [comment]
Yeah, it's been hard to get any movement from @leebyron to add maintainers to immutable-js. You can see our efforts thus far [here](https://github.com/immutable-js/immutable-js/issues/1689).

**Comment by mpereira** [comment]
Sorry that it's been such a hassle. Thanks for keeping on it!

**Comment by mpereira** [comment]
I sent him [a ping](https://twitter.com/mpereira/status/1370837649125163009) on Twitter.
